packages/perseus-editor/src/widgets/__tests__/interactive-graph-editor.test.tsx

```diff
@@ -43,6 +61,9 @@ describe("InteractiveGraphEditor", () => {
 
         render(
             <InteractiveGraphEditor {...baseProps} onChange={onChangeMock} />,
+            {
+                wrapper: RenderStateRoot,
+            },
```

Comment:
Needed to add `RenderStateRoot` because of the unique ID provider.

Reply:
:til: about `render()`'s `wrapper`!  Is it worth a `render` helper in this test file?